nuxt.config.js

```diff
@@ -37,7 +39,7 @@ export default {
   */
   buildModules: [
     // Doc: https://github.com/nuxt-community/eslint-module
-    '@nuxtjs/eslint-module',
+    // '@nuxtjs/eslint-module',
```

Comment:
Por que comentou aqui?

Reply:
Então, eu achei muito improdutivo no início, pois qualquer espaçamento fora do comum por exemplo dois espaços é um impedimento pra rodar o projeto. No início apareceram muitos warnings de aspas dupla, semicolon etc. dai eu decidi por hora comentar ele, e depois separar um tempo pra padronizar o código de acordo com o que ele aponta.

Reply:
Se vc acha que não precisamos dessa linha, não tenha medo. Deleta ela. É melhor que subir código comentado.

A não ser que vc tenha outra estratégia que eu ainda não entendi.

Reply:
Quando fiz o sample de login desabilitei também, fica bem improdutivo mesmo. 
Um espaçamento a mais depois do fim da linha já quebra o código. Futuramente acredito que vai sair plugins no vscode pra isso e ao salvar o arquivo já ira deixar nesse padrão, dai fica mais fácil. Hoje tenho isso no visual studio para algumas linguagens.

Concordo.